support-frontend/assets/pages/supporter-plus-thank-you/components/thankYouHeader/subheading.tsx

```diff
@@ -99,11 +115,12 @@ function Subheading({
 		isSignedIn,
 		identityUserType,
 	);
-
+	const isPending = paymentStatus === 'pending';
 	return (
 		<>
+			{isPending && pendingCopy()}
 			{subheadingCopy}
-			{!isGuardianAdLite && (
+			{!isGuardianAdLite && !isPending && (
```

Comment:
Presuming another link to manage account would be unavailable if in state pending?